docs/building.md

```diff
@@ -38,9 +38,26 @@ Building for a local `h` server
 -------------------------------
 
 The default settings file used is `chrome-dev.json` but it does not contain a 
-value for the `oauthClientId`. If you are running your own version of `h`, then 
-simply look up your oauth client id in the admin UI and add it to the config 
-file. `http://localhost:5000/admin/oauthclients`.
+value for the `oauthClientId`. If you are running your own version of `h` 
+simply create an oauth client ID in the admin UI for the browser extension.
+
+`http://localhost:5000/admin/oauthclients`.
```

Comment:
```suggestion
http://localhost:5000/admin/oauthclients.
```

Making this a plain URL rather than a quoted one means people can just click it when reading this file.